bundles/org.openhab.binding.freeboxos/src/main/java/org/openhab/binding/freeboxos/internal/discovery/FreeboxOsDiscoveryService.java

```diff
@@ -181,13 +181,13 @@ private void discoverHosts(FreeboxOsHandler localHandler, ThingUID bridgeUID, Li
                 thingDiscovered(builder.build());
             });
         } catch (PermissionException e) {
-            logger.warn("Missing permission to discover Hosts {}", e.getPermission());
+            logger.warn("Missing permission to discover Hosts: {}", e.getPermission());
         }
     }
 
-    private void discoverVM(VmManager vmManager, ThingUID bridgeUID, List<LanHost> lanHosts) throws FreeboxException {
+    private void discoverVM(ThingUID bridgeUID, List<LanHost> lanHosts) throws FreeboxException {
```

Comment:
The exception is caught within the method itself and not re-thrown, so it can be removed from the method signature.
```suggestion
    private void discoverVM(ThingUID bridgeUID, List<LanHost> lanHosts) {
```

Reply:
Fixed